src/envoy/http/jwt_auth/jwt.cc

```diff
@@ -333,8 +333,12 @@ bool Verifier::VerifySignatureRSA(EVP_PKEY *key, const EVP_MD *md,
                                   size_t signed_data_len) {
   bssl::UniquePtr<EVP_MD_CTX> md_ctx(EVP_MD_CTX_create());
 
-  EVP_DigestVerifyInit(md_ctx.get(), nullptr, md, nullptr, key);
-  EVP_DigestVerifyUpdate(md_ctx.get(), signed_data, signed_data_len);
+  if (EVP_DigestVerifyInit(md_ctx.get(), nullptr, md, nullptr, key) != 1) {
+    return false;
+  }
+  if (EVP_DigestVerifyUpdate(md_ctx.get(), signed_data, signed_data_len) != 1) {
+    return false;
```

Comment:
there should be a EVP_MD_CTX_destroy() of whatever was returned by EVP_MD_CTX_create().

Reply:
So far, EVP_MD_CTX_create() only allocate a piece of buffer. bssl::UniquePtr will free the buffer. So we are OK.

If EVP_MD_CTX_create() allocate embedded multiple buffers, then we have memory leak.